cpp/oneapi/dal/backend/primitives/distance/correlation_distance_dpc.cpp

```diff
@@ -0,0 +1,126 @@
+/*******************************************************************************
+* Copyright 2022 Intel Corporation
```

Comment:
The copyright notices we use after migration to  UXL:
```suggestion
* Copyright contributors to the oneDAL project
```